sgv2-docsapi/src/test/java/io/stargate/sgv2/docsapi/api/v2/example/ExampleResourceIntegrationTest.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.stargate.sgv2.docsapi.api.v2.example;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+import io.stargate.sgv2.docsapi.testprofiles.IntegrationTestProfile;
+import io.stargate.sgv2.docsapi.testresource.StargateTestResource;
+import org.junit.jupiter.api.Test;
+
+@QuarkusTest
+@QuarkusTestResource(StargateTestResource.class)
+@TestProfile(IntegrationTestProfile.class)
+class ExampleResourceIntegrationTest {
+
+  @Test
+  public void resourceLoads() {}
```

Comment:
Here is example of a test that does something:

```
    @Test
    public void keyspaceNotExisting() {
      String namespace = RandomStringUtils.randomAlphanumeric(16);
      String collection = RandomStringUtils.randomAlphanumeric(16);
      String body =
          """
              {
                "name": "%s"
              }
              """
              .formatted(collection);

      given()
          .contentType(ContentType.JSON)
          .body(body)
          .header(Constants.AUTHENTICATION_TOKEN_HEADER_NAME, "")
          .when()
          .post(BASE_PATH, namespace)
          .then()
          .statusCode(404)
          .body("code", is(404))
          .body(
              "description",
              is("Unknown namespace %s, you must create it first.".formatted(namespace)));
    }
```

Reply:
What's the value of `BASE_PATH`?

Reply:
Nevermind, I saw your other example in #1840